app/controllers/users.js

```diff
@@ -0,0 +1,24 @@
+const UsersController = module.exports;
+
+const { validationResult } = require('express-validator');
+
+const UsersService = require('../services/users');
```

Comment:
We name the variables in camelCase no PascalCase, please check this convention in all parts in the project